docs/production-deployment/cloud/nexus/using-nexus.mdx

```diff
@@ -0,0 +1,88 @@
+---
+id: using-nexus
+slug: /cloud/nexus/using-nexus
+title: Using Temporal Nexus
+description: Discover how to implement and use Nexus within Temporal workflows, including creating Nexus Services, Endpoints, and cross-team collaboration.
+sidebar_label: Using Temporal Nexus
+tags:
+  - temporal cloud
+  - nexus services
+  - Workflow orchestration
+  - cross-team workflows
+  - API endpoints
+keywords:
+  - Nexus Services in Temporal
+  - Workflow orchestration
+  - API endpoints
+  - cross-team collaboration
+  - cross-Namespace workflows
+  - Nexus API registry
+  - service-to-service communication
+---
+
+
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus API Registry. Nexus Endpoints include service documentation that enables others to use the Nexus Services provided by a Nexus Endpoint, from caller Workflows.  
+
+Temporal has built-in Nexus machinery that performs the low-level Nexus RPC operations on the wire and provides an integrated Temporal SDK experience to create Nexus Services in a handler Worker and use them from a caller Workflow via a Nexus Endpoint.
```

Comment:
I suggested a [Nexus RPC](https://github.com/temporalio/documentation/pull/3046#discussion_r1758147442) term for the glossary.